projects/storefrontstyles/scss/components/product/carousel/_carousel.scss

```diff
@@ -208,12 +208,18 @@
         background-color: var(--cx-color-background);
       }
     }
-    .indicators button[disabled] .cx-icon {
```

Comment:
I think we should leave the `button[disabled]` case in here with `[aria-disabled]` so there is no breaking change. We can leave a TODO comment to clean it up later.

Reply:
@Zeyber fixed;